README: Clarify steps to add a route

mac-kan opened the pull request:
Clarify possible steps of how to add a route to enable communication between the IOC and a TwinCAT Controller.

mac-kan committed:
README: Clarify steps to add a route

Clarify possible steps of how to add a route to enable communication
between the IOC and a TwinCAT Controller.

tboegi reviewed (commented):
Line 18: a route needs to be setup
May be "an ADS route needs to be setup" ?

2. This is for "old" systems running Windows.
I need to dig out the path for TC/BSD (which I think is the future)

Then: There is a 4. option for at least TC/BSD: The PLC has a web server.
Which allows to add routes in a mouse-ish way.
However, we can merges "as is" and put another PR on top

mac-kan committed:
README: Correct repository address

The repository name has been changed to twincat-ads. Update the README
with the correct address.

mac-kan commented:
Small update to add "ADS route" and correct the repository address.
